### ISSUE TITLE
User information is leaked in tournaments/tree

### ISSUE THREAD

**Issue by Plotogandr** [Plotogandr/AdessoKicker#21]
This includes hashed passwords, emails and pretty much every attribute available to user

**Comment by JanSchneider1** [comment]
> 
> 
> This includes hashed passwords, emails and pretty much every attribute available to user

This basically applies everywhere, where user data is used

**Comment by Plotogandr** [comment]
That's not the case, only if the user object directly is used as thymeleaf renders the HTML server side before sending it to the user.

**Comment by JanSchneider1** [comment]
> 
> 
> That's not the case, only if the user object directly is used as thymeleaf renders the HTML server side before sending it to the user.

Yeah, youre right, I just wanted to point out that we are using the user object a few times to iterate through the user base (like I did in _profile.html)
